snippets/current/src/set-up-anoncreds.ts

```diff
@@ -0,0 +1,49 @@
+import type { InitConfig } from '@aries-framework/core'
+
+const config: InitConfig = {
+  label: 'docs-agent-nodejs',
+  walletConfig: {
+    id: 'wallet-id',
+    key: 'testkey0000000000000000000000000',
+  },
+}
+
+// start-section-1
+import { Agent } from '@aries-framework/core'
+import { agentDependencies } from '@aries-framework/node'
+import { AskarModule } from '@aries-framework/askar'
+import { ariesAskar } from '@hyperledger/aries-askar-nodejs'
+import { IndyVdrAnonCredsRegistry } from '@aries-framework/indy-vdr'
+
+import { anoncreds } from '@hyperledger/anoncreds-nodejs'
+import { AnonCredsModule } from '@aries-framework/anoncreds'
+import { AnonCredsRsModule } from '@aries-framework/anoncreds-rs'
+
+const agent = new Agent({
+  config,
+  dependencies: agentDependencies,
+  modules: {
+    // Register the Askar module on the agent
+    // This is included as we need a wallet on our agent
+    // `indySdk` can also be used
```

Comment:
I'm not sure you can use Indy SDK with AnonCreds RS (we should definitely not promote it I think)

Reply:
@TimoGlastra Any suggestion for the CI failing? Without any registry it errors, so I could add a `// @ts-ignore` but that seems quite bad inside an example.

Reply:
I think we can add the indy-vdr one in that case, but I do want to have documentation pointing to all registries available (maybe using those cards again like we have with wallet / storage), and document that indy vdr is an example, but you could set it up with any anoncreds registry you want. And that if they want to use the indy vdr one, they need to set up the library correctly first.